source/appModules/windowsinternal_composableshell_experiences_textinput_inputapp.py

```diff
@@ -130,3 +233,19 @@ def event_nameChange(self, obj, nextHandler):
 		if obj.UIAElement.cachedAutomationID not in ("TEMPLATE_PART_ExpressionFullViewItemsGrid", "TEMPLATE_PART_ClipboardItemIndex", "CandidateWindowControl"):
 			ui.message(obj.name)
 		nextHandler()
+
+	def chooseNVDAObjectOverlayClasses(self, obj, clsList):
+		if isinstance(obj, UIA):
+			if obj.role == controlTypes.ROLE_LISTITEM and (
+				(
+					obj.parent.UIAAutomationId in (
+						"ExpandedCandidateList",
+						"TEMPLATE_PART_AdaptiveSuggestionList",
+					)
+					and obj.parent.parent.UIAAutomationId == "IME_Candidate_Window"
+				)
+				or obj.parent.UIAAutomationId == "IME_Candidate_Window"

```

Comment:
```suggestion
				or obj.parent.UIAAutomationId in ("IME_Candidate_Window", "IME_Prediction_Window")

```
In Microsoft pinyin automation id is "IME_Prediction_Window"